pkg/security/x509.go

```diff
@@ -33,11 +34,14 @@ const (
 	validFrom     = -time.Hour * 24
 	maxPathLength = 1
 	caCommonName  = "Cockroach CA"
+	tenantsOU     = "Tenants"
 )
 
 // newTemplate returns a partially-filled template.
 // It should be further populated based on whether the cert is for a CA or node.
-func newTemplate(commonName string, lifetime time.Duration) (*x509.Certificate, error) {
+func newTemplate(
+	commonName string, orgUnits []string, lifetime time.Duration,
```

Comment:
Was there any reason you decided against a signature `(commonName string, lifetime time.Duration, orgUnits ...string)`? If not, that seems preferable, avoiding all of the random `nil` parameters you're passing in at most callers.

Reply:
good idea. done